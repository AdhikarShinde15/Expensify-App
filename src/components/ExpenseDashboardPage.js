import React from "react";

const ExpenseDashboardPage = () => (
    <div>This is from Dashboard Page</div>
);

export default ExpenseDashboardPage;