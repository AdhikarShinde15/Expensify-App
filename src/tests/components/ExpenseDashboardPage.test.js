import React from "react";
import ExpenseDashboardPage from "../../components/ExpenseDashboardPage";
import { shallow } from "enzyme";

test('Should Render Expense DashBoard Correctly', () => {
    const wrapper = shallow(<ExpenseDashboardPage/>);
    expect(wrapper).toMatchSnapshot();
})