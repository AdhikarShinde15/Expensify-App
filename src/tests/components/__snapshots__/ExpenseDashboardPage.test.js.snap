// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Render Expense DashBoard Correctly 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
